misk-redis/src/main/kotlin/misk/redis/RealRedis.kt

```diff
@@ -302,6 +304,46 @@ class RealRedis(
     jedisPool.resource.use { jedis -> jedis.publish(channel, message) }
   }
 
+  override fun zadd(
+    key: String,
+    score: Double,
+    member: ByteString,
+    options: Set<String>): Long {
```

Comment:
should this be a `Set< ZAddOptions>`?

Reply:
I thought not exposing the callers to that might be better. It is an internal enum. We ultimately need `ZAddParams`. 
The other option was to have that argument be `ZAddParams`.
I am open to suggestions if folks think `Set< ZAddOptions>` or `ZAddParams` is better. Let me know.